src/components/sync-summary/SyncSummary.jsx

```diff
@@ -35,10 +114,60 @@ class SyncSummary extends React.Component {
                     fullWidth={true}
                 >
                     <DialogContent>
-                        <ExpansionPanel defaultExpanded>
+                        {response.map((responseElement, i) => (
+                            <ExpansionPanel
+                                defaultExpanded={response.length === 1}
+                                className={classes.expansionPanel}
+                                key={`row-${i}`}
+                            >
+                                <ExpansionPanelSummary expandIcon={<ExpandMoreIcon />}>
+                                    <Typography className={classes.expansionPanelHeading1}>
+                                        {`${responseElement.instance.name} (${
+                                            responseElement.instance.url
+                                        })`}
+                                    </Typography>
+                                    <Typography className={classes.expansionPanelHeading2}>
+                                        {`Status: ${responseElement.status}`}
+                                    </Typography>
+                                </ExpansionPanelSummary>
+
+                                <ExpansionPanelDetails className={classes.expansionPanelDetails}>
+                                    <Typography variant={"overline"}>Summary</Typography>
+                                </ExpansionPanelDetails>
+
+                                <ExpansionPanelDetails className={classes.expansionPanelDetails}>
+                                    {SyncSummary.buildSummaryTable([
+                                        ...responseElement.report.typeStats,
+                                        { type: "Total", ...responseElement.stats },
```

Comment:
Is it easy / makes sense to highlight the final total row with a background color?

Reply:
This is for @adrianq :)

I guess making it bold will suffice.

Reply:
Although we all love diaphanous interface some styling would be highly appreciated here I agree ;) Anyway, let's keep it like for now. We can rethink it a bit as part of https://github.com/EyeSeeTea/metadata-synchronization/issues/42